docs/topics/api/categories.rst

```diff
@@ -0,0 +1,125 @@
+==========
+Categories
+==========
+
+.. note::
+
+    These APIs are experimental and are currently being worked on. Endpoints
+    may change without warning.
+
+.. _category-list:
+
+Categories are defined by a name, a slug, a type and an application. Slugs are
+only guaranteed to be unique for a given ``app`` and ``type`` combination, and
```

Comment:
Actually, I don't think it'd really be that different, my brain was slow reading it I think. Leave it be, it reads perfectly this time.